package.json

```diff
@@ -8,7 +8,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
test script can be removed if it is not being used.

Reply:
I wrote about this in my Journal haha I knew I needed to get rid of one of them. I'll do that in the next PR